### PR TITLE
Multichannel

### PR DESCRIPTION
**Pull request by AlicjaSzu**
I want to merge this change because...

**PR intended to be tested with API branch:** feature/multichannel-multicurrency

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Translated strings are extracted.
1. [ ] Number of API calls is optimized.
1. [ ] The changes are tested.
1. [ ] Data-test are added for new elements.
1. [ ] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://feature-multichannel-multicurrency.api.saleor.rocks/graphql/

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -191,9 +193,7 @@ const Routes: React.FC = () => {
                 component={ProductSection}
               />
               <SectionRoute
-                permissions={[
-                  PermissionEnum.MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES
-                ]}
+                permissions={[PermissionEnum.MANAGE_PRODUCTS]}
```
Comment (orzechdev) [inline src/index.tsx:196]: Should it be changed? I see it is the path to product types, why does it require `MANAGE_PRODUCTS` permission?

File: src/index.tsx [hunk 2]
```diff
@@ -233,9 +233,7 @@ const Routes: React.FC = () => {
                 component={NavigationSection}
               />
               <SectionRoute
-                permissions={[
-                  PermissionEnum.MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES
-                ]}
+                permissions={[PermissionEnum.MANAGE_PRODUCTS]}
```
Comment (orzechdev) [inline src/index.tsx:236]: And here similarly? `MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES` sounds to be better for this section.

File: src/index.tsx (hunk 1 above)
Comment (dominik-zeglen) [reply src/index.tsx:196]: Looks like faulty rebase

**Review by dominik-zeglen — APPROVED** [review]

**Comment by dominik-zeglen** [comment]
Closing it to reopen with correct CI data

**Pull request closed without merging**